src/__tests__/standalone-syntax-test.js

```diff
@@ -133,3 +133,87 @@ test("standalone with syntax set by extension", t => {
 
   t.end()
 })
+
+test("standalone with path to custom parser", t => {
+  const config = {
+    rules: {
+      "block-no-empty": true,
+    },
+  }
+
+  standalone({
+    config,
+    syntax: `${fixturesPath}/custom-parser`,
+    code: ".foo { width: 200px; }\n.bar {}",
+    formatter: stringFormatter,
+  }).then(({ output }) => {
+    const strippedOutput = chalk.stripColor(output)
+    t.equal(typeof output, "string")
+    t.ok(strippedOutput.indexOf("2:6") !== -1)
+    t.ok(strippedOutput.indexOf("block-no-empty") !== -1)
+    t.end()
+  }).catch(t.end)
+})
+
+test("standalone with path to custom syntax", t => {
+  const config = {
+    rules: {
+      "block-no-empty": true,
+    },
+  }
+
+  standalone({
+    config,
+    syntax: `${fixturesPath}/custom-syntax`,
+    code: ".foo { width: 200px; }\n.bar {}",
+    formatter: stringFormatter,
+  }).then(({ output }) => {
+    const strippedOutput = chalk.stripColor(output)
+    t.equal(typeof output, "string")
+    t.ok(strippedOutput.indexOf("2:6") !== -1)
+    t.ok(strippedOutput.indexOf("block-no-empty") !== -1)
+    t.end()
+  }).catch(t.end)
+})
+
+test("standalone with npm-installed custom parser", t => {
+  const config = {
+    rules: {
+      "block-no-empty": true,
+    },
+  }
+
+  standalone({
+    config,
+    syntax: "postcss-safe-parser",
+    code: ".foo { width: 200px; }\n.bar {}",
+    formatter: stringFormatter,
+  }).then(({ output }) => {
+    const strippedOutput = chalk.stripColor(output)
+    t.equal(typeof output, "string")
+    t.ok(strippedOutput.indexOf("2:6") !== -1)
+    t.ok(strippedOutput.indexOf("block-no-empty") !== -1)
+    t.end()
+  }).catch(t.end)
+})
+
+test("standalone with npm-installed custom syntax", t => {
```

Comment:
I think both these `with npm-installed` tests can be removed, thereby removing the need to have these installed as `devDependencies`. As long as the code uses `getModulePath()` (see comment below), we can be confident that it will find npm dependencies.